homework/list.js

```diff
@@ -1,44 +1,48 @@
 'use strict';
 
-(function() {
-
-  var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
-
-  function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
-  }
-
-  function onNoteOpen(event) {
-    if (event.target.classList.contains('note-title')) {
+(function(exports) {
+  
+  var ToDoListManager = function(){
+    this._listNoteContent = [];
+    this._wrapper = document.querySelector('#note-list-wrapper');
+  };
+  
+  ToDoListManager.prototype = {
+    start(){
+      this.fetchList()
+      .then((function(data) {
+        this.updateList(data);
+        this.drawList();
+        this.preloadFirstNote();
+      }).bind(this));
+      window.addEventListener('click', (function(event) {
```

Comment:
It's a convention to prefix `on` to the corresponding function. So it should be renamed as `onClick` actually.